Gemfile.lock

```diff
@@ -114,61 +116,59 @@ GEM
       rb-fsevent (~> 0.9, >= 0.9.4)
       rb-inotify (~> 0.9, >= 0.9.7)
       ruby_dep (~> 1.2)
-    loofah (2.0.3)
+    loofah (2.4.0)
+      crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
-    mail (2.6.6)
-      mime-types (>= 1.16, < 4)
-    method_source (0.8.2)
-    mime-types (3.1)
-      mime-types-data (~> 3.2015)
-    mime-types-data (3.2016.0521)
-    mini_mime (0.1.4)
-    mini_portile2 (2.3.0)
-    minitest (5.10.3)
+    mail (2.7.1)
+      mini_mime (>= 0.1.1)
+    method_source (0.9.2)
+    mini_mime (1.0.2)
+    mini_portile2 (2.4.0)
+    minitest (5.13.0)
     mongo (2.4.3)
       bson (>= 4.2.1, < 5.0.0)
     mongoid (6.2.1)
       activemodel (~> 5.1)
       mongo (>= 2.4.1, < 3.0.0)
     multi_json (1.12.2)
-    nio4r (2.1.0)
-    nokogiri (1.8.1)
-      mini_portile2 (~> 2.3.0)
+    nio4r (2.5.2)
+    nokogiri (1.10.7)
+      mini_portile2 (~> 2.4.0)
     pipetree (0.1.1)
-    public_suffix (3.0.0)
+    public_suffix (4.0.1)
     puma (3.10.0)
-    rack (2.0.3)
-    rack-proxy (0.6.2)
+    rack (2.0.8)
+    rack-proxy (0.6.5)
       rack
-    rack-test (0.7.0)
+    rack-test (1.1.0)
       rack (>= 1.0, < 3)
-    rails (5.1.4)
-      actioncable (= 5.1.4)
-      actionmailer (= 5.1.4)
-      actionpack (= 5.1.4)
-      actionview (= 5.1.4)
-      activejob (= 5.1.4)
-      activemodel (= 5.1.4)
-      activerecord (= 5.1.4)
-      activesupport (= 5.1.4)
+    rails (5.1.7)
+      actioncable (= 5.1.7)
+      actionmailer (= 5.1.7)
+      actionpack (= 5.1.7)
+      actionview (= 5.1.7)
+      activejob (= 5.1.7)
+      activemodel (= 5.1.7)
+      activerecord (= 5.1.7)
+      activesupport (= 5.1.7)
       bundler (>= 1.3.0)
-      railties (= 5.1.4)
+      railties (= 5.1.7)
       sprockets-rails (>= 2.0.0)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.0.3)
-      loofah (~> 2.0)
-    railties (5.1.4)
-      actionpack (= 5.1.4)
-      activesupport (= 5.1.4)
+    rails-html-sanitizer (1.3.0)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Possible XSS vulnerability in rails-html-sanitizer (CVE-2015-7580)](https://app.codacy.com/manual/rfizzle/destiny/pullRequest?prid=5063452)